tslearn/docs/examples/plot_sax_mindist_knn.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""
+1-NN with SAX + MINDIST
+=======================
+
+This example presents a comparison performs kNN with k=1, using MINDIST from
```

Comment:
Suggestion for improvement:

This example presents a comparison between k-Nearest Neighbor runs with k=1.
It compares the use of:
- MINDIST (see [1]) on SAX-transformed data;
- Euclidean distance on raw time series.

The comparison is based on test accuracy computed for several benchmark datasets.

[1] Cite the paper here